.github/workflows/main.yml

```diff
@@ -16,18 +16,16 @@ jobs:
           node-version: 18
       - name: Build Packages
         run: |
-          npm i
+          npm i --include=dev
```

Comment:
why do we need this?

Reply:
In my local build if I don't include dev dependencies I can't run tests, cause karma is a dev dependency

Reply:
I think we don't need it on the CI